packages/ios_platform_images/lib/ios_platform_images.dart

```diff
@@ -66,6 +66,7 @@ class _FutureMemoryImage extends ImageProvider<_FutureMemoryImage> {
   /// See [ImageProvider.load].
   // TODO(jmagman): Implement the new API once it lands, https://github.com/flutter/flutter/issues/103556
   @override
+  // ignore: deprecated_member_use
```

Comment:
What should we be doing instead, and when? It would be good to have a TODO with an issue capturing what we should be changing this code to, and what version of Flutter needs to be on `stable` before we can change.

Reply:
The issue is https://github.com/flutter/flutter/issues/103556

>and what version of Flutter needs to be on stable before we can change.

That is TBD. I can't actually land the PR required to add/deprecate this functionality until this repo is green, plus rolling into google3. That could take another week or it could take another two months.

As part of this change I will have a migration guide, but that also can't be submitted until we have a version which is contingent on rolling into g3

Reply:
Got it; just a TODO to reimplement once possible with a reference to the issue link works then.

Reply:
Is the one on L67 good enough? 😄

Reply:
🤦🏻 

In my defense there's an entire line between this line and that comment!

Oh, and now that I see line 65 I see why I'm getting deja-vu here. Should we just put this `ignore` on every other line of the file just to be safe? 😬